modules/sdk/node-build/configuration/getProjectWebContextPath.mjs

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs/promises';
+
+export default async function getProjectWebContextPath() {
+	const bnd = await fs.readFile('./bnd.bnd', 'utf-8');
+
+	const parts = 
+		bnd
+			.split('\n')
+			.find(line => line.startsWith('Web-ContextPath:'))
+			.split(':');
```

Comment:
This potentially could error here if it doesn't fine the line since you can't .split false

Reply:
done